file-formats/edcc/type/zif_aff_edcc_v1.intf.abap

```diff
@@ -0,0 +1,608 @@
+    BEGIN OF ty_message,
+      "! <p class="shorttext">Message Type</p>
+      "! Message type
+      "! $required
+      message_type                TYPE c LENGTH 20,
+      "! <p class="shorttext">Tax Authority Document Type</p>
+      "! Tax authority document type
+      "! $required
+      tax_authority_document_type TYPE c LENGTH 20,
+    END OF ty_message.
+  "! <p class="shorttext">Tax Authority Message Types</p>
+  "! Tax authority message types
+  TYPES ty_messages TYPE SORTED TABLE OF ty_message WITH UNIQUE KEY message_type.
+  TYPES:
+    "! <p class="shorttext">Additional Tax Authority Table</p>
+    "! Additional tax authority table
+    BEGIN OF ty_tax_authority_table,
+      "! <p class="shorttext">Table Name</p>
+      "! Table name
+      "! $required
+      table_name TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_tax_authority_table.
+  "! <p class="shorttext">Additional Tax Authority Tables</p>
+  "! Additional tax authority tables
+  TYPES ty_tax_authority_tables TYPE SORTED TABLE OF ty_tax_authority_table WITH UNIQUE KEY table_name.
+  "! <p class="shorttext">Position Number</p>
+  "! Position number
+  TYPES ty_position_number      TYPE n LENGTH 2.
+  "! <p class="shorttext">Result Process</p>
+  "! Result process
+  TYPES ty_result_process       TYPE c LENGTH 10.
+  "! <p class="shorttext">Field Value</p>
+  "! Field value
+  TYPES ty_field_value          TYPE c LENGTH 40.
+  "! <p class="shorttext">Check Number</p>
+  "! Check number
+  TYPES ty_check_number         TYPE n LENGTH 5.
+  "! <p class="shorttext">Description</p>
+  "! Description
+  TYPES ty_long_description     TYPE c LENGTH 120.
+  TYPES:
+    "! <p class="shorttext">Existence Check</p>
+    "! Assigned existence check
+    BEGIN OF ty_existence_check,
+      "! <p class="shorttext">Check Id</p>
+      "! Check id
+      "! $required
+      check_id       TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Field Name</p>
+      "! Field name
+      "! $required
+      field_name     TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Field Value</p>
+      "! Field value
+      "! $required
+      field_value    TYPE ty_field_value,
+      "! <p class="shorttext">Result Process</p>
+      "! Result process
+      "! $required
+      result_process TYPE ty_result_process,
+    END OF ty_existence_check.
+  TYPES:
+    "! <p class="shorttext">Status Check</p>
+    "! Status check
+    BEGIN OF ty_status_check,
+      "! <p class="shorttext">Check Id</p>
+      "! Check id
+      "! $required
+      check_id             TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Sequence Number</p>
+      "! Sequence number
+      "! $required
+      sequence_number      TYPE ty_sequence_number,
+      "! <p class="shorttext">Source Field Name 1</p>
+      "! Source field name 1
+      "! $required
+      source_field_name_1  TYPE zif_aff_types_v1=>ty_object_name_30,
```

Comment:
For the existence check, you only have `field_name` and `field_value`. Is the term `source` important for the status check? Furthermore, I wondered wether 
```suggestion
      "! <p class="shorttext">First Source Field Name</p>
      "! First source field name
      "! $required
      first_source_field_name  TYPE zif_aff_types_v1=>ty_object_name_30,
```
(or without `source`) is better as numbers in the names are quite unusal.